build-tools/packages/build-tools/package.json

```diff
@@ -42,45 +42,45 @@
 		"@fluid-tools/version-tools": "workspace:~",
 		"@manypkg/get-packages": "^2.2.2",
 		"async": "^3.2.6",
-		"cosmiconfig": "^8.3.6",
-		"date-fns": "^2.30.0",
+		"cosmiconfig": "^9.0.0",
+		"date-fns": "^3.6.0",
 		"debug": "^4.4.3",
 		"detect-indent": "^6.1.0",
 		"find-up": "^7.0.0",
 		"fs-extra": "^11.3.2",
 		"glob": "^7.2.3",
 		"globby": "^11.1.0",
-		"ignore": "^5.3.2",
+		"ignore": "^7.0.5",
 		"json5": "^2.2.3",
 		"lodash.isequal": "^4.5.0",
 		"multimatch": "^5.0.0",
 		"picocolors": "^1.1.1",
-		"picomatch": "^2.3.1",
+		"picomatch": "^4.0.3",
 		"picospinner": "^2.1.0",
-		"rimraf": "^4.4.1",
+		"rimraf": "^6.1.2",
 		"semver": "^7.7.3",
 		"sort-package-json": "1.57.0",
 		"ts-deepmerge": "^7.0.3",
-		"type-fest": "^2.19.0",
 		"typescript": "~5.4.5",
 		"yaml": "^2.8.1"
 	},
 	"devDependencies": {
 		"@biomejs/biome": "~1.9.4",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools-bin": "npm:@fluidframework/build-tools@~0.49.0",
-		"@fluidframework/eslint-config-fluid": "^6.1.0",
+		"@fluidframework/eslint-config-fluid": "^8.1.0",
 		"@types/async": "^3.2.25",
 		"@types/fs-extra": "^11.0.4",
-		"@types/glob": "^7.2.0",
+		"@types/glob": "^8.1.0",
 		"@types/lodash.isequal": "^4.5.8",
 		"@types/mocha": "^10.0.10",
 		"@types/node": "^22.19.1",
-		"@types/rimraf": "^2.0.5",
+		"@types/rimraf": "^3.0.2",
```

Comment:
Seems like we can just do away with this one now: https://www.npmjs.com/package/@types/rimraf